### ISSUE TITLE
[BUG] Unable to login with User role, Dashboard not seen.

### ISSUE THREAD

**Issue by basavarajvs** [polonel/trudesk#153]
**Is this a BUG REPORT or FEATURE REQUEST?**:

- [X] BUG 
- [ ] FEATURE

**What happened**:
Unable to login with user created with User role.

**What did you expect to happen**:
Login should be possible for users with any role.

**How to reproduce it (as minimally and precisely as possible)**:
1. Login as Admin
2. Create a new user and assign role as User.
3. Logout Admin
4. Login with newly created user credentials
5. The URL redirects to the dashboard and does not proceed further.
The dashboard is not visible.

**Anything else we need to know?**:
After changing the new user's role to Admin the newly created user is able to login with credentials

**Environment**:

- Trudesk Version:1.0.7-beta
- OS (e.g. from /etc/os-release): OS-X
- Node.JS Version:
- MongoDB Version:
- Is this hosted on cloud.trudesk.io: No

**Comment by polonel** [comment]
I can confirm this is an issue. Working on a solution

**Comment by polonel** [comment]
I want to clarify. I removed the dashboard from users a while back due to the data displayed in the dashboard is global. (Users would see stats from the entire system)

This is a flaw that will be addressed after the agent system is done. It will allow me to load customized data based on the teams the agent is assigned to. And the dashboard for users would show for their respected Customer Groups.

I have fixed the login issue for users but as it was previously, users will be redirected to the tickets view.
